FilmsApp/View/ListController/Extension.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+import UIKit
+extension UIView {
+ 
+ func anchor (top: NSLayoutYAxisAnchor?, left: NSLayoutXAxisAnchor?, bottom: NSLayoutYAxisAnchor?, right: NSLayoutXAxisAnchor?, paddingTop: CGFloat, paddingLeft: CGFloat, paddingBottom: CGFloat, paddingRight: CGFloat, width: CGFloat, height: CGFloat, enableInsets: Bool) {
+ var topInset = CGFloat(0)
+ var bottomInset = CGFloat(0)
+ 
+ if #available(iOS 11, *), enableInsets {
+ let insets = self.safeAreaInsets
+ topInset = insets.top
+ bottomInset = insets.bottom
+ 
+ print("Top: \(topInset)")
+ print("bottom: \(bottomInset)")
+ }
+ 
+ translatesAutoresizingMaskIntoConstraints = false
+ 
+ if let top = top {
+ self.topAnchor.constraint(equalTo: top, constant: paddingTop+topInset).isActive = true
+ }
+ if let left = left {
+ self.leftAnchor.constraint(equalTo: left, constant: paddingLeft).isActive = true
+ }
+ if let right = right {
+ rightAnchor.constraint(equalTo: right, constant: -paddingRight).isActive = true
+ }
+ if let bottom = bottom {
```

Comment:
Fix alignment

Reply:
I have to change alignment in this block of code, or I have to fix my code with this alignments?

Reply:
Just highlight this function and use a shortcut 'control + i'